tests/ImageSharp.Benchmarks/General/BasicMath/ModuloPowerOfTwoVariable.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Attributes.Jobs;
+
+namespace SixLabors.ImageSharp.Benchmarks.General.BasicMath
+{
+    [LongRunJob]
+    public class ModuloPowerOfTwoVariable
+    {
+        private readonly int value = 42;
+
+        private readonly int m = 32;
+
+        [Benchmark(Baseline = true)]
+        public int Standard()
+        {
+            return this.value % this.m;
+        }
+
+        [Benchmark]
+        public int Bitwise()
+        {
+            return ImageMaths.ModuloP2(this.value, this.m);
+        }
+
+        // RESULTS:
```

Comment:
I need to make sure I add the results like this when benchmarking. Really good idea.